### ISSUE TITLE
Accessibility - Invisible Text

### ISSUE THREAD

**Issue by T-T-Nguyen** [StatCan/WellbeingCheckUAT#254]
Feedback from Accessibility:

The invisible text is still not referencing the “contact us section of the app. This will provide users a much easier/accessible way of contacting statistics Canada if necessary.

What are the options @Jiandong-Guo? I recall you stating thay hyperlink are not possible.

**Comment by Jiandong-Guo** [comment]
The current text is 

The 'Results' section of the Well-being Check contains summaries of your responses to the questionnaire. There is a snapshot of your overall self-rated feelings, and three bar charts that display a summary of your feelings by location, activity, and who you are with. If your screen reader cannot read these results, we would be happy to send them to you in a different format. _Please send us an email with your name and postal code to infostats@canada.ca (e-mail address must be manually entered). Please also ensure your name and postal code is the same as what was entered in the first well-being check you completed to submit this request.",_

My suggestion is to change the "please send us....." to something like "please reference the contact us screen to send a request to us with your name and postal code that is the same as what was entered in the first well-being check you completed",

**Comment by T-T-Nguyen** [comment]
@Jiandong-Guo 


Final English:
The 'Results' section of the Well-being Check contains summaries of your responses to the questionnaire. There is a snapshot of your overall self-rated feelings, and three bar charts that display a summary of your feelings by location, activity, and who you are with. If your screen reader cannot read these results, we would be happy to send them to you in a different format. Please send us an email with your name and postal code to infostats@canada.ca (e-mail address must be manually entered).  Please also ensure your name and postal code is the same as what was entered in the first well-being check you completed to submit this request. For additional information or more ways to reach us, please refer to the “Contact us” section of the application.

Final French:
La section des résultats du Bilan bien-être contient un résumé de vos réponses au questionnaire. On y trouve un aperçu de vos sentiments généraux autoévalués, ainsi que trois graphiques à barres affichant un sommaire de vos sentiments selon votre emplacement, vos activités, et les personnes avec qui vous êtes. Si votre lecteur d’écran n’arrive pas à lire ces résultats, nous pouvons vous les envoyer dans un autre format. Veuillez nous écrire à l’adresse infostats@canada.ca en indiquant votre nom et votre code postal (l’adresse de courriel doit être entrée manuellement). Veuillez également vous assurer que votre nom et votre code postal sont identiques à ceux que vous avez entrés dans le premier Bilan bien-être que vous avez rempli pour présenter cette demande. Pour obtenir de plus amples informations ou afin d’obtenir d’autres moyens pour nous contacter, veuillez vous diriger à la section « Contactez-nous » de l’application.

**Comment by Jiandong-Guo** [comment]
Changed.  will see in next release

**Comment by T-T-Nguyen** [comment]
@Jiandong-Guo 
Here is the updated text to include the name change:

Final English:
The 'Results' section of the StatCan Attitude app contains summaries of your responses to the questionnaire. There is a snapshot of your overall self-rated feelings, and three bar charts that display a summary of your feelings by location, activity, and who you are with. If your screen reader cannot read these results, we would be happy to send them to you in a different format. Please send us an email with your name and postal code to infostats@canada.ca (e-mail address must be manually entered).  Please also ensure your name and postal code are the same as what was entered in the questionnaire. For additional information or more ways to reach us, please refer to the “Contact us” section of the application.

Final French:
La section « Résultats » de l’application StatCan Attitude contient un résumé de vos réponses au questionnaire. On y trouve un aperçu de vos sentiments généraux autoévalués, ainsi que trois graphiques à barres affichant un sommaire de vos sentiments selon votre emplacement, vos activités, et les personnes avec qui vous êtes. Si votre lecteur d’écran n’arrive pas à lire ces résultats, nous pouvons vous les envoyer dans un autre format. Veuillez nous écrire à l’adresse infostats@canada.ca en indiquant votre nom et votre code postal (l’adresse de courriel doit être entrée manuellement). Lorsque vous présenter cette demande, veuillez également vous assurer que votre nom et votre code postal sont identiques à ceux que vous avez entrés dans le questionnaire. Pour obtenir de plus amples informations ou afin d’obtenir d’autres moyens pour nous contacter, veuillez vous diriger à la section « Contactez-nous » de l’application.

**Comment by Jiandong-Guo** [comment]
done

**Comment by Jiandong-Guo** [comment]
[voice.zip](https://github.com/StatCan/WellbeingCheckUAT/files/6611221/voice.zip)
@T-T-Nguyen Please check the voice for this accessibility text

**Comment by Jiandong-Guo** [comment]
This is under french device setting to read both french and english
[Voix_210616.zip](https://github.com/StatCan/WellbeingCheckUAT/files/6665465/Voix_210616.zip)
